### PR TITLE
Rename TS Known Issues execution pipeline

### PR DESCRIPTION
**Pull request by rkodev**
Renames TS Execution Known Issues Pipeline

### CODE REVIEW

File: TypeScriptV1Tests/TsDependecies/tsconfig.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	"compilerOptions": {
 		"module": "CommonJS",
 		"esModuleInterop": true,
-		"target": "ES2018",
```
Comment (baywet) [inline TypeScriptV1Tests/TsDependecies/tsconfig.json:5]: I'm curious to the rationale being this?

Comment (rkodev) [reply TypeScriptV1Tests/TsDependecies/tsconfig.json:5]: Aligning it with https://github.com/microsoftgraph/msgraph-sdk-typescript/blob/dev/tsconfig.json

**Review by fey101 — APPROVED** [review]
LGTM

File: TypeScriptV1Tests/TsDependecies/tsconfig.json (hunk 1 above)
Comment (rkodev) [reply TypeScriptV1Tests/TsDependecies/tsconfig.json:5]: The SDK is targeting `ES2015` the tests are also set to compile and be compatible with ES2015

Comment (baywet) [reply TypeScriptV1Tests/TsDependecies/tsconfig.json:5]: shouldn't it be aligned with kiota libraries that target es2017? https://github.com/microsoft/kiota-typescript/blob/0ab7f4e705fa2fc7e17bbef0fcb624334bdeada5/packages/serialization/json/tsconfig.es.json#L5

This probably also impacts our SDK final size. (granted that core still also targets es2015)

**Review by baywet — APPROVED** [review]

**Pull request merged**